app/models/product.rb

```diff
@@ -6,5 +6,14 @@ class Product < ApplicationRecord
   validates :name, presence: true
   validates :price, presence: true, numericality: { greater_than: 0 }
   validates :position, presence: true
+
   acts_as_list
+
+  scope :purchasable, -> {
+    where(disabled: false)
+  }
```

Comment:
【好み】これくらいなら改行なしで書くことが多いかなーと思いました(あと改行する場合は do 〜 end 使うようにの sgcop なかったかな？)
```suggestion
  scope :purchasable, -> { where(disabled: false) }
```

Reply:
[138bc83](https://github.com/kk0000-kk/sakura-market/pull/24/commits/138bc831456c76e93884e73341d096605b06665e)